### ISSUE TITLE
How would I go about modifying bounds.max_y & bounds.min_y?

### ISSUE THREAD

**Issue by Commandthealien** [Builderb0y/BigGlobe#79]
In Fabric version 1.21.1, I keep getting recurring message telling me to check my logs.

Message:
`[Big Globe/Scripting]: Caught exception from builderb0y.bigglobe.columns.scripted.ColumnScript$ColumnToIntScript$Holder: java.lang.NullPointerException: Cannot invoke "builderb0y.bigglobe.columns.scripted.VoronoiDataBase$Generated$Base_SkylandCell_136.get_bounds()" because the return value of "builderb0y.bigglobe.columns.scripted.ScriptedColumn$Generated_106.get_bigglobe_overworld_skyland_186()" is null; Check your logs for more info.
`

While viewing the [logs](https://paste.ee/p/EjzMi) on lines 1341-1343 it mentions: 
```
[11:20:25] [Big Globe Worker Thread/ERROR]: Script source was:
1:	SkylandBounds bounds = `bigglobe:overworld/skyland`.bounds
2:	bounds != null && bounds.max_y > bounds.min_y ? floorInt(bounds.max_y) + 1 : 16x8000_0000U
```

I am unsure if there is any solution as there is nothing to modify in the config folder.

**Comment by Builderb0y** [comment]
You don't. This is a bug in my own code which I'll fix in the next version. It's weird that it's only becoming a problem now though, since that bug has been there for a very long time.

I also see a lot of errors with distant horizons. I'll fix those too.

**Comment by Commandthealien** [comment]
Good to know and I'm glad light was shed on other issues as well.

**Comment by Builderb0y** [comment]
Both issues fixed in V4.8.4.
